net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild

```diff
@@ -0,0 +1,300 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit autotools eutils fixheadtails qmail user
+
+HOMEPAGE="http://www.inter7.com/index.php?page=vpopmail"
+DESCRIPTION="Collection of programs to manage virtual email on Qmail servers"
+SRC_URI="mirror://sourceforge/${PN}/${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
+IUSE="clearpasswd ipalias maildrop mysql postgres spamassassin"
+REQUIRED_USE="mysql? ( !postgres )"
+
+DEPEND="virtual/qmail
+	maildrop? ( mail-filter/maildrop )
+	mysql? ( dev-db/mysql-connector-c:0= )
+	postgres? ( dev-db/postgresql:=[server] )
+	spamassassin? ( mail-filter/spamassassin )"
+RDEPEND="${DEPEND}"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-5.4.9-access.violation.patch
+	"${FILESDIR}"/${PN}-lazy.patch
+	"${FILESDIR}"/${PN}-vpgsql.patch
+	"${FILESDIR}"/${PN}-double-free.patch
+	"${FILESDIR}"/${PN}-5.4.33-vdelivermail-add-static.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-those-vfork-instances-that-do-more-than-exec.patch
+	"${FILESDIR}"/${PN}-5.4.33-remove-unneeded-forward-declaration.patch
+	"${FILESDIR}"/${PN}-5.4.33-clean-up-calling-maildrop.patch
+	"${FILESDIR}"/${PN}-5.4.33-fix-S-tag-in-case-spamassassin-changed-the-file-size.patch
+	"${FILESDIR}"/${PN}-5.4.33-strncat.patch
+)
+DOCS=(
+	ChangeLog
+	doc/AUTHORS
+	doc/FAQ
+	doc/INSTALL
+	doc/README.activedirectory
```

Comment:
If a wildcard doesnt work here, you could use some bash to shorten the list at least.

Reply:
No, because I now get that now:

 * checking miscfile checksums ;-) ...      [ ok ]
…/net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild: line 46: no match: doc/README.*
…/net-mail/vpopmail/vpopmail-5.4.33-r4.ebuild: line 49: no match: doc/doc_html/*